ctapipe/containers.py

```diff
@@ -503,17 +564,51 @@ class R1CameraContainer(Container):
     Storage of r1 calibrated data from a single telescope
     """
 
+    event_type = Field(EventType.UNKNOWN, "type of event", type=EventType)
+    event_time = Field(NAN_TIME, "event timestamp")
+
     waveform = Field(
         None,
         (
             "numpy array containing a set of images, one per ADC sample"
             "Shape: (n_pixels, n_samples)"
         ),
     )
+
+    pixel_status = Field(
+        None,
+        "Array of pixel status values, see PixelStatus for definition of the values",
+        dtype=np.uint8,
+    )
+
+    first_cell_id = Field(
+        None,
+        "Array of first cell ids of the readout chips. Only used by LST and SST.",
+        dtype=np.uint32,
+    )
+
+    module_hires_local_clock_counter = Field(
```

Comment:
not a great name, but it's also bad in the document...  Not totally clear why the word "local" is there. I guess local is that the time reference is module-dependent? Maybe at least mention what this means in the help string so we know how to use it later.  `hires` is also just about the storage format, so could be dropped.  This could just be `module_clock_counter`, with a help string saying something like "count of clock ticks since a locally defined reference within the camera module" (if that is indeed what it is...)

Reply:
Is it worth it to deviate from the term in the data model document? I'd say we need a very strong reason to do that and then we'd have to add references like "note: this is called module_hires_local_clock_counter in the data model" every where. 

I also don't much like the chosen name, but at least in this case I preferred to just follow the name in the data model instead of trying to invent a better one.

Reply:
yeah, maybe stick with the name and just add a better help string for now. We can always update the name when the model evolves.